eng-wasm/derive/src/eth_contract.rs

```diff
@@ -0,0 +1,154 @@
+use std::fs::File;
```

Comment:
I copied the implementation of the `eth_contract` macro to this new submodule. i did not otherwise change the implementation, except a small change related to breaking changes in `syn` 1.0